src/main/java/Main.java

```diff
@@ -0,0 +1,59 @@
+import moves.MovesStrategy;
+import moves.Paper;
+import moves.Rock;
+import moves.Scissors;
+
+import java.util.Scanner;
+import java.util.logging.Logger;
+
+public class Main {
+    public static void main(String[] args) {
+        System.out.println("ROCK PAPER SCISSORS");
+        MovesStrategy firstPlayer;
+        MovesStrategy secondPlayer;
+        do {
```

Comment:
move this block to chooseMove method
        do {
            firstPlayer =  chooseMove();
        }while (firstPlayer == null);